src/index.html

```diff
@@ -5,8 +5,415 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Righteous&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header-logo">
+        <img 
+          src="./images/logo.png" 
+          alt="Moyo"
+          class="header-logo"
+        >
+      </a>
+      <nav class="header-nav">
+        <ul class="header-nav__list">
+          <li class="header-nav__item">
+            <a href="#apple" class="header-nav__link is-active">Apple</a>
+          </li>
+          <li class="header-nav__item">
+            <a href="#samsung" class="header-nav__link">Samsung</a>
+          </li>
```

Comment:
```suggestion
          <li class="header-nav__item">
            <a href="#apple" class="header-nav__link is-active">Apple</a>
          </li>

          <li class="header-nav__item">
            <a href="#samsung" class="header-nav__link">Samsung</a>
          </li>
```

add blank line between multiple sibling blocks

Reply:
i'm fix it, please check my work))